tests/rptest/redpanda_cloud_tests/observe_test.py

```diff
@@ -59,15 +59,23 @@ def cluster_alerts(self, rule_groups):
 
     @cluster(num_nodes=0, check_allowed_error_logs=False)
     def test_cloud_observe(self):
-        self.logger.debug("Here we go")
+        self.logger.debug("Starting cloud observe test")
 
         rule_groups = self.load_grafana_rules()
-
         alerts = self.cluster_alerts(rule_groups.data.groups)
+        high_priority_alerts = []
 
         for alert in alerts:
-            self.logger.warn(
-                f'alert firing for cluster: {alert.labels.grafana_folder} / {alert.labels.alertname}'
-            )
-
-        assert len(alerts) == 0
+            alert_message = f"alert firing for cluster: {alert.labels.grafana_folder} / {alert.labels.alertname}"
+            if "high priority" in alert_message.lower():
```

Comment:
can yo invert this conditional statement please

we should filter out "low priority", and append everything else to high priority list to fail on if not empty. That way if someone adds a new alert without a priority, or someone changes the text for high priority alerts we still catch them. e.g., we want to catch the case where "high priority - foo bar" gets changed to "sev 1 - foo bar". Currently if that happens they just get bundled into low priority warn messages and we go on oblivious